### PR TITLE
[MIG] SQS connector migration odoo 17.0

### PR DESCRIPTION
**Pull request by rejamen**

### CODE REVIEW

**Review by awisky — COMMENTED** [review]
nice

File: sqs/static/description/sqs_menu_icon.png
Comment (awisky) [inline sqs/static/description/sqs_menu_icon.png:1]: Niceeeee Icon 🥇

**Review by awisky — CHANGES_REQUESTED** [review]
Nice Icon @r

**Comment by rejamen** [comment]
@awisky I deleted pycache files.

**Review by awisky — APPROVED** [review]

**Pull request merged**